selection/index_selection_evaluation.py

```diff
@@ -73,11 +74,16 @@ def _setup_config(self, config):
                                          table_generator.columns)
         self.workload = Workload(query_generator.queries, self.database_name)
 
+        if 'pickle_workload' in config and config['pickle_workload'] is True:
+            pickle_filename = f"benchmark_results/workload_{config['benchmark_name']}_{len(self.workload.queries)}_queries.pickle"
+            pickle.dump(self.workload, open(pickle_filename, "wb"))
+
     def _run_algorithms(self, config_file):
         with open(config_file) as f:
             config = json.load(f)
         self._setup_config(config)
         self.db_connector.drop_indexes()
+        self.db_connector.set_random_seed()
```

Comment:
Add comment
# For deterministic statistics, as ANALYZE uses random samples for large tables